src/test/java/com/google/cloud/tools/gradle/appengine/core/DeployAllTaskTest.java

```diff
@@ -92,26 +93,85 @@ public void testDeployAllAction() throws AppEngineException, IOException {
 
     deployAllTask.deployAllAction();
 
-    assertTrue(deployConfig.getDeployables().contains(appYaml));
-    assertTrue(deployConfig.getDeployables().contains(cronYaml));
-    assertTrue(deployConfig.getDeployables().contains(dispatchYaml));
-    assertTrue(deployConfig.getDeployables().contains(dosYaml));
-    assertTrue(deployConfig.getDeployables().contains(indexYaml));
-    assertTrue(deployConfig.getDeployables().contains(queueYaml));
-    assertFalse(deployConfig.getDeployables().contains(invalidYaml));
-    verify(deploy).deploy(deployConfig);
+    verify(deploy).deploy(deployCapture.capture());
+    DeployConfiguration captured = deployCapture.getValue();
+    assertTrue(captured.getDeployables().contains(appYaml));
+    assertTrue(captured.getDeployables().contains(cronYaml));
+    assertTrue(captured.getDeployables().contains(dispatchYaml));
+    assertTrue(captured.getDeployables().contains(dosYaml));
+    assertTrue(captured.getDeployables().contains(indexYaml));
+    assertTrue(captured.getDeployables().contains(queueYaml));
+    assertFalse(captured.getDeployables().contains(invalidYaml));
```

Comment:
Do we have the side effect check? `originalDeployConfig.getDeployables == ['original-deployable.yaml']`

Reply:
never mind I found it.